### DISCUSSION TITLE
Welcome to mimotion Discussions!

### DISCUSSION THREAD

**Discussion by TonyJiangWJ** [TonyJiangWJ/mimotion#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yichuang-space** [comment]
小米运动里面步数更新了，但是无法同步到微信里呀，注销重新注册还一样呢

**Comment by WGooold** [comment]
一刷 号就登录不上去了 是被ban了吗

**Comment by zkdf007** [comment]
添加名为 AES_KEY 的Secret变量，请自行创建一个长度为16个字符的字符串作为密钥
需要注意，为了避免出现问题，密钥请不要使用中文。请自行确保长度为16个字符。
当你账号过多或者需要自行登录时，请配置AES_KEY，以便开启自动保存。配置AES_KEY后，将自动将密钥加密保存到当前仓库，请一定保存好自己的AES密钥，不要泄露。
同时请确保配置正确的PAT密钥，否则将无法自动保存提交到仓库。
当配置了 AES_KEY 之后，因为每个人的仓库里面到会保存一份 encrypted_tokens.data，更新代码会被覆盖。为了避免数据丢失，请提前备份，在更新完成后将它重新提交到仓库中，然后再触发workflow。

菜鸟对于这一段不是很明白。大佬能否讲的简单一点！！

**Reply by TonyJiangWJ** [reply]
见README 看不懂你问AI 让他帮你解释
